code/modules/antagonists/nukeop/equipment/nuclearbomb.dm

```diff
@@ -531,7 +531,7 @@
 
 /obj/machinery/nuclearbomb/proc/really_actually_explode(off_station)
 	Cinematic(get_cinematic_type(off_station),world,CALLBACK(SSticker,/datum/controller/subsystem/ticker/proc/station_explosion_detonation,src))
-	INVOKE_ASYNC(GLOBAL_PROC,PROC_REF(KillEveryoneOnZLevel), get_virtual_z_level())
+	INVOKE_ASYNC(GLOBAL_PROC, GLOBAL_PROC_REF(KillEveryoneOnZLevel), get_virtual_z_level())
```

Comment:
515 fix, nice, but probably a bad PR to include this

Reply:
Probably. I don't see the fuss, however. It's related enough.

Reply:
I'll add it to the changelog